app/macos/Podfile.lock

```diff
@@ -100,4 +100,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce7dbe26c78bfc7ba46736094c1e2d25982870fa
 
-COCOAPODS: 1.13.0
```

Comment:
Please revert this to ```1.13.0``` and would suggest upgrading cocoapods on your local (if using) to minimum ```1.13.0``` as previous version has bugs with MacOS and Xcode latest versions.

Reply:
It was by mistake pushed. I will revert this changes.